lib/src/queries/devices/device_details_fragment/query.fragment.ast.gql.dart

```diff
@@ -469,6 +469,242 @@ const DeviceDetails = _i1.FragmentDefinitionNode(
                             ]))
                       ]))
                 ])),
+            _i1.InlineFragmentNode(
```

Comment:
It's worth discussing this in our code sync. 

Sentiment in StackOverflow seems mixed:
[[Should I commit generated code in Flutter/Dart to VCS](https://stackoverflow.com/questions/56110386/should-i-commit-generated-code-in-flutter-dart-to-vcs)](https://stackoverflow.com/a/70346661)

It could be a good argument to separate the SDK out into a separate parts with one containing the schema + generated code only.